### DISCUSSION TITLE
Feature Request: StashDB scene count versus Stash scene Count (Per Performer)

### DISCUSSION THREAD

**Discussion by str0keR** [stashapp/stash#5277] [Ideas]
When viewing scenes for a performer, it would be nice to see a visual indication of how many scenes you have from what's available in StashDB?

So for example, Emma Bugg - I have 33 scenes

- 32 of them have StashIDs
- 1 scene has no StashID
- StashDB has 74 scenes

It would be useful to see an indicator to show:

- I have 32/74 scenes for Emma Bugg
- 1 scene is not in StashDB

and then maybe if I click on the indicator and be presented with a list of what's missing from my library compared to what's in StashDB

**Comment by DogmaDragon** [comment]
Direct integration with SlashDB is unlikely. But there are plugins/external tools that achieve something similar. 

- https://github.com/feederbox826/stashlist
- https://github.com/MinasukiHikimuna/MidnightRider-Stash/tree/main/plugins/CompleteTheStash#complete-the-stash
